assets/js/theme/custom/show-secondary-product-card-image.js

```diff
@@ -0,0 +1,64 @@
+export default function showSecondaryProductCardImage() {
+    const selectors = {
+        figureWrapper: '.js-card-figure-wrapper',
+        primaryImage: '.js-primary-image',
+        secondaryImage: '.js-secondary-image',
+        prevButton: '.js-prev-thumbnail-btn',
+        nextButton: '.js-next-thumbnail-btn',
+    }
+
+    const toggleVisibility = (image) => {
+        const firstImage = image.querySelector(selectors.primaryImage)
+        const secondImage = image.querySelector(selectors.secondaryImage)
+        if (secondImage) {
+            firstImage.classList.toggle('is-hidden')
+            secondImage.classList.toggle('is-hidden')
+        }
+    }
+
+    const figureWrappers = document.querySelectorAll(selectors.figureWrapper)
+    if (figureWrappers.length > 0) {
+        figureWrappers.forEach((figure) => {
```

Comment:
You don't need to check if `figureWrappers.length` is greater than 0 if you are iterating with forEach